kong/tools/faker.lua

```diff
@@ -33,6 +33,9 @@ Faker.FIXTURES = {
     { name = "API TESTS 6", public_dns = "cors1.com", target_url = "http://mockbin.com" },
     { name = "API TESTS 7", public_dns = "cors2.com", target_url = "http://mockbin.com" },
 
+    { name = "API TESTS 8 (dns)", public_dns = "dns1.com", target_url = "http://127.0.0.1:7771" },
+    { name = "API TESTS 9 (dns)", public_dns = "dns2.com", target_url = "http://localhost:7771" },
```

Comment:
Yeah it's nice it's enough to know it's for the DNS. We'll move them out of the faker and to their own respective tests later.